cmd/kubelet/app/options/options.go

```diff
@@ -87,6 +83,7 @@ type KubeletFlags struct {
 // a kubelet. These can either be set via command line or directly.
 type KubeletServer struct {
 	KubeletFlags
+	ContainerRuntimeOptions
```

Comment:
I named it in the original PR but changed it to be consistent with other fields in the struct. Will switch it back.

> Also, would it make sense for these to be subordinate to the KubeletFlags object, rather than directly on the KubeletServer, given that they all come in as flags-only?

Make sense. We still need to pass the container runtime options as a whole to kubelet, but I think that's fine. Done.